geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala

```diff
@@ -185,7 +187,8 @@ object AccumuloDataStoreParams {
   val recordThreadsParam   = new Param("recordThreads", classOf[Integer], "The number of threads to use for record retrieval", false, 10)
   val writeThreadsParam    = new Param("writeThreads", classOf[Integer], "The number of threads to use for writing records", false, 10)
   val looseBBoxParam       = new Param("looseBoundingBox", classOf[java.lang.Boolean], "Use loose bounding boxes - queries will be faster but may return extraneous results", false, true)
-  val statsParam           = new Param("collectStats", classOf[java.lang.Boolean], "Toggle collection of statistics", false, true)
```

Comment:
It still worked on a store I had registered, but collect stats was not selected when i went back to the data store page so it will change the configuration.